pkg/parser/parser.go

```diff
@@ -861,3 +881,47 @@ func parseString(s string) (string, string, error) {
 
 	return s[:i], s[i+1:], nil
 }
+
+func StartAlignTo(start int64, alignTo string) (int64, error) {
+	var newDate time.Time
+
+	if IsDigit(alignTo[0]) {
+		re := regexp.MustCompile(`^[0-9]+`)
+		alignTo = re.ReplaceAllString(alignTo, "")
+	}
```

Comment:
nit: this panics if `len(alignTo) == 0`, I think we can just do the replace without checking:
```suggestion
	re := regexp.MustCompile(`^[0-9]+`)
	alignTo = re.ReplaceAllString(alignTo, "")
```